copy_to_modules/Styla/controllers/admin/Styla_PathsTabController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+class Styla_PathsTabController extends oxAdminDetails
+{
+    
+    /**
+     * @var object|Styla_Util
+     */
+    protected $_oStylaUtil;
+    
+    /**
+     * Current class template name.
+     * @var string
+     */
+    protected $_sThisTemplate = 'Styla_pathsAdminTab.tpl';
+    
+    function __construct()
+    {
+        parent::__construct();
+        $this->_oStylaUtil = oxNew('Styla_Util');
+    }
+
+    /**
+     * save
+     * -----------------------------------------------------------------------------------------------------------------
+     * Saves configuration parameters.
+     *
+     * @throws oxSystemComponentException
+     */
+    public function save()
+    {
+        $oConfig = oxRegistry::getConfig();
+        $sOxid = $this->getEditObjectId();
+        $iLang = $this->_iEditLang;
+        $aParams = $oConfig->getRequestParameter('editval');
+        //checkboxhandling
+        if(isset($aParams['styla_home'])){
+            $aParams['styla_home'] = 1;
+            $aParams['oxid'] = 'styla_home';
+        }
+        else{
+            $aParams['styla_home'] = 0;
+        }
+        $oModel = oxNew('Styla_Paths');
+        if ($sOxid != "-1") {
+            $oModel = oxNew('Styla_Paths');
+            $oModel->loadInLang($iLang,$sOxid);
+        }
+        $oModel->assign($aParams);
+        $oModel->save();
+        parent::save();
+        $this->setEditObjectId($oModel->getId());
+        //no seo for landingpage
+        if(!$aParams['styla_home']){
```

Comment:
why no seo for start page? the seo should be added same as for the magazine part?